epilepsy12/models_folder/case.py

```diff
@@ -64,7 +64,20 @@ class Case(TimeStampAbstractBaseClass, UserStampAbstractBaseClass, HelpTextMixin
         blank=True,
     )
     nhs_number = models.CharField(  # the NHS number for England and Wales
-        "NHS Number", unique=True, blank=True, null=True, max_length=10
+        "NHS Number",
+        unique=True,
+        blank=True,
+        null=True,
+        max_length=10,
+        help_text="This is the NHS number for England and Wales. It is used to identify the patient in the audit.",
+    )
+    unique_reference_number = models.CharField(
```

Comment:
As mentioned above, I think we could use a [UniqueConstraint](https://docs.djangoproject.com/en/5.1/ref/models/constraints/#uniqueconstraint) across NHS number and URN rather than enforcing uniqueness for URNs at the form cleaner level

Reply:
I agree